examples/How_to_handle_rate_limits.ipynb

```diff
@@ -31,59 +31,108 @@
    "source": [
     "## Default rate limits\n",
     "\n",
-    "As of Jan 2023, the default rate limits are:\n",
```

Comment:
Let's update the content here with the information from our new rate limits docs: https://platform.openai.com/docs/guides/rate-limits?context=tier-free

Maybe we can mainly just link to the doc so that we don't have to worry about keeping this up to date as well.

Reply:
Ok I removed the table and put a link instead!